l10n/en/mozorg/home-old.ftl

```diff
@@ -0,0 +1,31 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/ ('en-US', 'en-CA', 'en-GB', 'de' and 'fr' excluded)
```

Comment:
Heads up: renaming an existing FTL file will break translations for the old home page. It's probably better to create a new FTL file for the redesigned home page, and leave the old FTL file intact

Reply:
I have that in my notes to fix up! :) thanks for the sharp eyes here